### DISCUSSION TITLE
Licht Sensor zeigt keine Reaktion

### DISCUSSION THREAD

**Discussion by knebelu** [beabel/radiowecker#44] [Fragen & Antworten / Q&A] [answered]
Ich habe jetzt den Lichtsensor KY-018 von AZ -so wie in der verlinkten Anleitung- beschrieben angebaut. Den Ausgang S des Sensors habe ich mit dem auf der Leiterplatte mit A0 bezeichneten Kontakt verlötet. Im Code ist der PIN 36 für den Licht Sensor definiert. Ich bin nicht sicher ob diese Verbindung so richtig ist? Änderungen am Code habe ich bisher keine vorgenommen.  Der Lichtsensor zeigt in dieser Konstellation leider keinerlei Wirkung. Habe ich da etwas übersehen?

**Comment by Hotzi57** [comment, marked as answer]
Klingt alles richtig. Du musst die Helligkeit auf 0% runter regeln, dann wird er aktiv!

**Reply by knebelu** [reply]
Ja! Danke, das ist es. Helligkeit auf 0% und schon klappt es.
